unischedule/android/app/src/main/AndroidManifest.xml

```diff
@@ -13,15 +13,16 @@
             android:theme="@style/LaunchTheme"
             android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
             android:hardwareAccelerated="true"
-            android:windowSoftInputMode="adjustResize">
+            android:windowSoftInputMode="adjustResize"
+            android:screenOrientation="portrait"> <!-- Línea agregada para bloquear la orientación en modo retrato -->
```

Comment:
Great work @iGotty ! Quick question: will this work on iOS?

Reply:
@jsurrea Not really!! jeje

 The current implementation for Android does not directly translate to iOS due to platform-specific differences. For iOS, we need to handle the screen orientation configuration within the Xcode project settings and possibly adjust some Flutter Dart code to ensure compatibility.